docs/configuration.md

```diff
@@ -233,6 +233,9 @@ ga_tracking: "UA-1234567-89,G-1AB234CDE5"
 By default, the navigation and search include normal [pages](https://jekyllrb.com/docs/pages/).
 You can also use [Jekyll collections](https://jekyllrb.com/docs/collections/) which group documents semantically together.
 
+{: .warning }
+> Collection folders always start with an underscore (`_`), e.g. `_tests`. You won't see your collections if you skip the prefix.
```

Comment:
LGTM. (I'd have written "omit" rather than "skip", but it's clear enough as is.)

Reply:
I completely missed that - it was written by githubs co-pilot 😂